Say Their Names/Source/Controller/Petitions/PetitionsViewCode.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  PetitionsUI.swift
+//  Say Their Names
+//
+//  Created by Kyle Lee on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Petitions
+final class PetitionsViewCode: UIView {
+    
+    private lazy var petitionsLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Petitions"
+        label.textAlignment = .center
+        
+        return label
+    }()
+    
+    init() {
+        super.init(frame: .zero)
+        setupSelf()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        setupSubviews()
```

Comment:
`layoutSubviews` is called quite often, and the setup needs to happen only once. So it's better to put it in `setupSelf` method instead